install-nginx.sh

```diff
@@ -20,7 +20,10 @@ then
             ;;
     esac
 else
-    CONFIG_FILE=/etc/nginx/router_config_local.yaml
+    GENERATE_CONFIG_FILE=/usr/local/bin/generate_config_router_mesh
```

Comment:
I don't see that the `GENERATE_CONFIG_FILE`, `TEMPLATE_FILE_PLUS`, and `TEMPLATE_FILE` environment variables are used anywhere.

Do we need them?